databases/relational-databases.md

```diff
@@ -171,87 +135,23 @@ Also briefly review: [LucidChart: What is a Database Model?](https://www.lucidch
-  - [Difference between MySQL and PostgreSQL](https://techdifferences.com/difference-between-mysql-and-postgresql.html)
-
-### Common Mistakes / Misconceptions
-
-SQL is a standard language used with multiple relational databases (Postgres, MySQL, SQLite, SQLServer, etc.).
-
-SQL statements are just text that we send to a database. The database then executes the SQL statements, and sends the results back to us.
-
-Relational databases help ensure data integrity because they enforce a schema, which places restrictions on the data we insert. Designing a schema usually only needs to happen when you have a new type of data that you want to insert into the DB.
-
-Speeding up queries in relational databases requires understanding database indexing and schema design. Typically, relational databases support many types of advanced indexing strategies. Using these can improve the execution speed of your SQL from (say) hours to milliseconds.
-
-The most prevalent type of database index is a B-Tree, which is a version of a Binary Search Tree optimized for disk access.
-
-Schema design is a deep topic, so it is okay if you don't grok<sup>1</sup> everything at once.
-
-Make sure you don't slow down a database for other users. This can happen if you execute a query that takes up all system resources. (Typically, faster queries use fewer system resources.)
-
-Because you must define a schema to store data in a relational database, some developers think that they are harder to use than "schema-less" databases like MongoDB. However, taking the effort to use a schema for your data can drastically reduce the likelihood of bugs as your project matures, so many developers choose relational DBs.
-
 ### Guided Practice
 
 - [Install Postgres](/databases/installing-postgresql.md) locally
 - Connect to Postgres with the commandline client, then:
   - Create a _database_
   - Create a _table_
-  - Create another table that is different but related, and that shares at least one column with the first table created (i.e. create a table that can be logically `JOIN`ed to the first table)
+  - Create another table that is different but related, and that shares at least one column with the first table created (i.e. create a table that can be logically connected to the first table)
   - Add rows of data to the new table(s)
   - Execute a few queries to retrieve data, as guided by instructor or volunteers. (While this will likely be review of content from the [SQL](./sql.md) lesson plan, practice will help sharpen these skills.)
 - Go through the course [Databases Course: Querying Relational Databases (145 min)](https://teamtreehouse.com/library/querying-relational-databases). In this course the concepts of Set Theory and Database Normalization are introduced. You will also learn about Database Keys and Table Relationships. Without keys, databases would not be able to function. Table Relationship help us to describe the way that one table links or relates to another.
```

Comment:
One of the big asks of the issue was to make the guided practice guided with step-by-step instructions and it's mostly unchanged here. Can you fill in how to do each step? Feel free to pick some example data to illustrate the concepts such as an animals or pets table, or something else less dry than the usual customer examples of the SQL world.